db/schema.rb

```diff
@@ -11,208 +10,217 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141217071943) do
+ActiveRecord::Schema.define(version: 2018_06_14_210435) do
 
-  create_table "delayed_job_failures", force: true do |t|
-    t.string   "summary"
-    t.binary   "details",    limit: 2147483647
-    t.boolean  "read",                          default: false, null: false
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "delayed_job_failures", id: :serial, force: :cascade do |t|
```

Comment:
did all of these changes get recorded as migrations (e.g. adding the id column here and changing force to :cascade)?  does it matter?  travis-ci seems to run through the migrations

Reply:
The actual code is in the migrations and this just got auto-generated. Rails guidelines say we should check this into version control but it looks like the generated code depends on what database the developer used.